xclim/indices/_hydrology.py

```diff
@@ -279,3 +283,106 @@ def melt_and_precip_max(
+        Normalized Qp, which is the p th percentile of daily streamflow normalized by the median flow.
+
+    References
+    ----------
+    :cite:cts:`addor2018,Clausen2000`
+    """
+    qp = q.quantile(p, dim="time")
+    q_median = q.median(dim="time")
+    out = qp / q_median
+    out.attrs["units"] = "1"
+    return out
+
+
+@declare_units(q="[discharge]")
+def high_flow_frequency(
+    q: xr.DataArray, threshold_factor: int = 9, freq: str = "YS-OCT", **indexer
+) -> xr.DataArray:
+    """
+    High flow frequency.
+
+    Calculate the number of days in a given period with flows greater than a specified threshold. By default, the
+    period is the water year starting on 1st October and ending on 30th September, as commonly defined in North America.
+
+    Parameters
+    ----------
+    q : xr.DataArray
+        Daily streamflow data.
+    threshold_factor : int
+        Factor by which the median flow is multiplied to set the high flow threshold, default is 9.
+    freq : str, optional
+        Resampling frequency, default is 'YS-OCT' for water year starting in October and ending in September.
+    indexer
+        Indexing parameters to perform a temporal subset of the data.
+        It accepts the same arguments as :py:func:`xclim.indices.generic.select_time`.
+
+    Returns
+    -------
+    xr.DataArray
+        Calculated mean of high flow days per water year
+
+    References
+    ----------
+    :cite:cts:`addor2018,Clausen2000`
+    """
+    median_flow = q.median(dim="time")
+    threshold = threshold_factor * median_flow
+    sel = select_time(q, **indexer)
+    out = threshold_count(sel, ">", threshold, freq=freq)
+    return to_agg_units(out, q, "count")
+
+
+@declare_units(q="[discharge]")
+def low_flow_frequency(
+    q: xr.DataArray, threshold_factor: float = 0.2, freq: str = "YS-OCT", **indexer
+) -> xr.DataArray:
+    """
+    Calculate the number of days in a given period with flows lower than a specified threshold. By default, the period is the water year starting on 1st October and ending on 30th September, as commonly defined in North America.
+
+    Parameters
+    ----------
+    q : xr.DataArray
+        Daily streamflow data.
+    threshold_factor : float
+        Factor by which the mean flow is multiplied to set the low flow threshold, default is 0.2.
+    freq : str, optional
+        Resampling frequency, default is 'YS-OCT' for water year starting in October and ending in September.
+    indexer
+        Indexing parameters to perform a temporal subset of the data.
+        It accepts the same arguments as :py:func:`xclim.indices.generic.select_time`.
+
+    Returns
+    -------
+    xr.DataArray
+        Calculated mean of low flow days per water year.
+
+    References
+    ----------
+    :cite:cts:`Olden2003`
+    """
+    mean_flow = q.mean(dim="time")
+    threshold = threshold_factor * mean_flow
+    sel = select_time(q, **indexer)
+    out = threshold_count(q, "<", threshold, freq=freq)
```

Comment:
I imagine you want this?
```suggestion
    out = threshold_count(sel, "<", threshold, freq=freq)
```

Reply:
> I imagine you want this?

yes